changelogs/unreleased/2949-sseago

```diff
@@ -0,0 +1 @@
+Restore CRD Resource name to fix CRD wait functionality.
```

Comment:
@ashish-amarnath I believe this update is in line with what @nrb had suggested. If I misunderstood, or if more than a one-line summary is better, I can change it again.